libtiledbsoma/src/tiledbsoma/tiledbsoma

```diff
@@ -49,6 +49,8 @@
 #include "soma/column_buffer.h"
 #include "soma/soma_array.h"
 #include "soma/soma_collection.h"
+#include "soma/soma_column.h"
+#include "soma/soma_dimension.h"
```

Comment:
Looks like you remove these in a future PR. Might as well just not add them in to begin with.